src/reyna/Preferences/Preferences.cs

```diff
@@ -83,6 +86,37 @@ public bool OffChargeBlackout
             }
         }
 
+        public bool BatchUpload
+        {
+            get
+            {
+                return GetRegistryValue(BatchUploadKeyName, true);
+            }
+        }
+
+        public Uri BatchUploadUrl
+        {
+            get
+            {
+                var url = GetRegistryValue(BatchUploadUriKeyName, string.Empty);
```

Comment:
Just saw that we have that on Android as well, but why do we need to be able to set the URL for batch upload to anything but the default? Is that already for people using some sort of proxy?